eventheader_macros/src/enabled_generator.rs

```diff
@@ -71,9 +71,10 @@ impl EnabledGenerator {
                         self.tree2
                             .add_punct("&")
                             .push_span(provider_symbol_span)
-                            .add_ident(&String::from_iter(
-                                [PROVIDER_PTR_VAR_PREFIX, &provider_symbol_string].into_iter(),
-                            ))
+                            .add_ident(&String::from_iter([
+                                PROVIDER_PTR_VAR_PREFIX,
+                                &provider_symbol_string,
+                            ]))
```

Comment:
Note: Changes in `eventheader_macros` are just clippy warning fixes plus auto-fmt reformatting.